src/coverlet.core/Extensions/DependencyInjectionExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Coverlet.Core.Attributes;
+
+namespace coverlet.core.Extensions
+{
+    public static class DependencyInjectionExtensions
```

Comment:
when we'll merge https://github.com/tonerdo/coverlet/pull/566 we'll hide as internals...for now it's ok.

Reply:
Ok, for now it will stay public.